ocp_resources/datavolume.py

```diff
@@ -229,3 +244,36 @@ def _check_none_pending_status(self, failure_timeout=120):
         except TimeoutExpiredError:
             self.logger.error(f"{self.name} status is {sample}")
             raise
+
+    def is_garbage_collector_enabled_dv(self):
+        if self.exists:
+            dv_annotations = self.instance.metadata.get("annotations")
+            if dv_annotations:
+                return (
+                    dv_annotations.get("cdi.kubevirt.io/storage.deleteAfterCompletion")
+                    is not False
+                )
+        return True
```

Comment:
```suggestion
    def is_garbage_collector_enabled_dv(self):
        if self.exists:
            return self.instance.metadata.get("annotations", {}).get("cdi.kubevirt.io/storage.deleteAfterCompletion", False) is not False

```

`is_garbage_collector_enabled_dv` cannot return `True` if DV does not exist

Reply:
changed it